.goreleaser.yml

```diff
@@ -50,3 +50,34 @@ nfpms:
     formats:
       - deb
       - rpm
+    overrides:
+      deb:
```

Comment:
LGTM thanks, https://github.com/cloudflare/gokeyless/blob/master/Makefile#L58-L82 seems to roughly match the behavior this this command (what was previously used to use packages) does

Reply:
will have @mitalirawat double check

Reply:
lgtm. We should probably remove this file now but still discussing.
```
- src: pkg/gokeyless.sysv
        dst: /etc/init.d/gokeyless
```